README.md

```diff
@@ -1,104 +1,157 @@
-# pravega-samples
+# Pravega and Analytics Connectors Examples
 
-Sample applications for Pravega.
+This repository contains code samples to demonstrate how developers can work with 
+[Pravega](http://pravega.io). We also provide code samples to connect analytics 
+engines such as [Flink](https://flink.apache.org/) and
+[Hadoop](http://hadoop.apache.org/) with Pravega as a storage substrate for data 
+streams. 
 
-## Getting Started
+For more information on Pravega, we recommend to read the [documentation and the
+developer guide](http://pravega.io).
 
-### Building Pravega
+# Repository Structure
 
-Optional: This step is required only if you want to use a different version
-of Pravega than is published to maven central.
+This repository is divided into sub-projects (`standalone-examples`, `flink-examples`
```

Comment:
Are we thinking of moving `anomaly-detection` module under `flink-examples` (since it is not listed in the sub-projects list) ?

Reply:
While this is not decided yet, the `anomaly-detection` application will probably be placed in a separate folder called `scenarios` (after the reorganization of the repository). Please, have a look to the repository reorganization proposal in this [Wiki page](https://github.com/pravega/pravega-samples/wiki/Repository-Structure-Reorganization) and provide feedback if necessary.